system-tests/test/config_modules_spec.ts

```diff
@@ -39,11 +39,12 @@ describe('compiles config files using the native node import', () => {
   ;[
     // esbuild chokes on these kinds of projects (JS Config File + TSConfig that's out of range)
     // so this makes sure we're using the native node import
-    'config-cjs-and-esm/config-with-mjs-tsconfig-es5',
-    'config-cjs-and-esm/config-with-cjs-tsconfig-es5',
-    'config-cjs-and-esm/config-with-js-tsconfig-es5',
-    'config-cjs-and-esm/config-with-js-tsconfig-es3',
-    'config-cjs-and-esm/config-with-js-tsconfig-es2015',
+    // 'config-cjs-and-esm/config-with-mjs-tsconfig-es5',
```

Comment:
what's the reason these are commented out?

Reply:
Just wanted to run 1 test. I'll add them back in.